### ISSUE TITLE
Feature request: close project without save

### ISSUE THREAD

**Issue by scriptingsentinel** [OpenFunscripter/OFS#1]
Can you add a _close project_ (without saving) in the project menu?

I wanted to start a new project but couldn't unless I saved the project I had open and I didn't want to save the changes. I had to close OFS without saving first to get rid of my changes and then I had to restart and do a _save and close project_ since the last project is auto-loaded.

**Comment by Noise973** [comment]
I agree with scriptingentinel - the file load/save operations should all be in the File menu, and you should be able to bail out of unsaved changes.

The one thing I really need is "Home" and "End" commands - using the mouse to slide back to the beginning of a clip is a little annoying (and not always very accurate).

**Comment by OpenFunscripter** [comment]
I removed it from the "Project" menu and moved it below "Open project" in the "File" menu.
It's now just "Close project" and will close if there are no changes.
If there are unsaved changes it will show a yes/no dialog first before closing.

I also added "Go to the start/end" bindings but without default keys so they have to be set first before they can be used.

**Comment by Noise973** [comment]
Thanks! Great project, really useful!
